### ISSUE TITLE
Possible to extend the supported networks ? chainId.ts

### ISSUE THREAD

**Issue by re2005** [chnejohnson/vue-dapp#50]
Thanks for the amazing work done! Works like a charm.

I'm new to it, and I'm looking for a way to extent the default list of chains.
That is because you already store the chains and all that information inside the plugin store.

Is there any way to extend the current list ? without I'm having to keep track of the current chain outside of the plugin?

I'm taking about this file:
https://github.com/chnejohnson/vue-dapp/blob/main/src/constants/chainId.ts

Also using the provided method `displayChainName` would resolve correctly:

```
import { displayChainName } from 'vue-dapp';
```

Thanks!

**Comment by chnejohnson** [comment]
Thanks! Glad you like it.

Hmm...let me think about this problem. About `displayChainName` is as follows:
https://github.com/chnejohnson/vue-dapp/blob/9a948222a60622f8b339c50877ab41e10aabbcb1/src/utils/format.ts#L18-L24

**Comment by re2005** [comment]
I mean, way to extend this constant:

`https://github.com/chnejohnson/vue-dapp/blob/main/src/constants/chainId.ts`

Could pass to the directive as a new property:

```
<vd-board
    :chainIds="['1', '2']"
    :connectors="connectors"
/>
```

And that get merged to the default list.

**Comment by re2005** [comment]
Here we go, please consider this PR.

https://github.com/chnejohnson/vue-dapp/pull/53

Would love start contributing more to this project.

**Comment by chnejohnson** [comment]
Merged, thanks :)

**Comment by chnejohnson** [comment]
Published in v0.5.3
